frontend/src/features/projects/erp/forms/ExemptionEnhancedReferralCompleteForm.tsx

```diff
@@ -74,92 +74,126 @@ const ExemptionEnhancedReferralCompleteForm = ({
           field="clearanceNotificationSentOn"
         />
       </Form.Row>
-      <h3>Option 1: Transfer within the Greater Reporting Entity</h3>
-
-      <Form.Row>
-        <Form.Label column md={4}>
-          Date Transferred within the GRE
-          <TooltipIcon
-            toolTipId="onDateTransferredWithinGre"
-            toolTip={onTransferredWithinTheGreTooltip}
-          />
-        </Form.Label>
-        <FastDatePicker
-          outerClassName="col-md-2"
-          formikProps={formikProps}
-          disabled={isReadOnly}
-          field="transferredWithinGreOn"
-        />
-        {!isComplete && (
-          <div className="col-md-6">
-            <Button
-              disabled={isReadOnly || !formikProps.values.transferredWithinGreOn}
-              onClick={onClickGreTransferred}
-            >
-              Update Property Information
-            </Button>
-          </div>
-        )}
-      </Form.Row>
-      <h3>Option 2: Proceed to Surplus Properties List</h3>
-      <Form.Row>
-        <Form.Label column md={4}>
-          Request Received On
-        </Form.Label>
-        <FastDatePicker
-          outerClassName="col-md-2"
-          formikProps={formikProps}
-          disabled={isReadOnly}
-          field="requestForSplReceivedOn"
-        />
-      </Form.Row>
-      <Form.Row>
-        <Form.Label column md={4}>
-          SPL Addition Approved On
-        </Form.Label>
-        <FastDatePicker
-          outerClassName="col-md-2"
-          formikProps={formikProps}
-          disabled={isReadOnly}
-          field="approvedForSplOn"
-        />
-        {!isComplete && formikProps.values.statusCode !== ReviewWorkflowStatus.NotInSpl && (
-          <>
-            <Button
-              disabled={
-                isReadOnly ||
-                !formikProps.values.requestForSplReceivedOn ||
-                !formikProps.values.approvedForSplOn
-              }
-              onClick={() => {
-                setProceedToSpl(true);
-              }}
-            >
-              Proceed to SPL
-            </Button>
-            {formikProps.values.statusCode !== ReviewWorkflowStatus.NotInSpl && (
-              <>
-                <OrText>OR</OrText>
-                <Button disabled={isReadOnly} onClick={onClickNotInSpl}>
-                  Not Included in the SPL
-                </Button>
-              </>
-            )}
-          </>
-        )}
-      </Form.Row>
-      {!isComplete && (
+      {![
```

Comment:
Only show these sections when appropriate.  Generally don't show this here if currently in SPL.